### ISSUE TITLE
Getting  "Error: ENOENT: no such file or directory" in path : azure/msal-react/src/packageMetadata.ts

### ISSUE THREAD

**Issue by luan-dang-techlabs** [AzureAD/microsoft-authentication-library-for-js#6382]
### Core Library

MSAL.js (@azure/msal-browser)

### Core Library Version

3.0.2

### Wrapper Library

MSAL React (@azure/msal-react)

### Wrapper Library Version

2.0.2

### Public or Confidential Client?

Public

### Description

e.g. package.json

```
...
devDependencies: {
   "react-scripts": "5.0.1"
},
 scripts: {
  "start": "react-scripts start"
 }
...
...
```

I am getting this message because I don't have path **'src/'** when running ```npm start```

<img width="1205" alt="Screenshot 2023-08-22 at 11 41 44 AM" src="https://github.com/AzureAD/microsoft-authentication-library-for-js/assets/91907117/3699f162-ec6c-4390-96c8-21b647ddf1c5">

<img width="1231" alt="Screenshot 2023-08-22 at 11 45 02 AM" src="https://github.com/AzureAD/microsoft-authentication-library-for-js/assets/91907117/d17a2bdc-fb02-4ecf-bf7f-22db2c2cc07a">

### Identity Provider

Other

### Source

External (Customer)

**Comment by lalimasharda** [comment]
Hey @luan-dang-techlabs, are you seeing this error on install/build? Can you double check our guidance on [package installation](https://github.com/AzureAD/microsoft-authentication-library-for-js/blob/dev/lib/msal-react/README.md#installation)?

**Comment by luan-dang-techlabs** [comment]
> Hey @luan-dang-techlabs, are you seeing this error on install/build? Can you double check our guidance on [package installation](https://github.com/AzureAD/microsoft-authentication-library-for-js/blob/dev/lib/msal-react/README.md#installation)?

Sorry, I should have made it more clear.  I get this error when I start my local server like this:

```
 package.json 


{
...
devDependencies: {
   "react-scripts": "5.0.1"
},
 scripts: {
  "start": "react-scripts start"
 }
...
```

* when i start my local server like this:
```
 $ npm start
```

**Comment by lalimasharda** [comment]
Did you try building the packages locally before running the app? Maybe try `npm run build:package` like [here](https://github.com/AzureAD/microsoft-authentication-library-for-js/blob/908e97f68b3d3732750c09c80b1bd702f98c2949/samples/msal-react-samples/react-router-sample/package.json#L21C1-L22C1).

**Comment by luan-dang-techlabs** [comment]
> Did you try building the packages locally before running the app? Maybe try `npm run build:package` like [here](https://github.com/AzureAD/microsoft-authentication-library-for-js/blob/908e97f68b3d3732750c09c80b1bd702f98c2949/samples/msal-react-samples/react-router-sample/package.json#L21C1-L22C1).

I didn't, let give that a try.

**Comment by luan-dang-techlabs** [comment]
> > Did you try building the packages locally before running the app? Maybe try `npm run build:package` like [here](https://github.com/AzureAD/microsoft-authentication-library-for-js/blob/908e97f68b3d3732750c09c80b1bd702f98c2949/samples/msal-react-samples/react-router-sample/package.json#L21C1-L22C1).
> 
> I didn't, let give that a try.

Even though I had those message, authenticating is working and local server works fine. The issue lies with ```Failed to parse source map from ``` message showing up. Did something change with the way 'source-map' was created ?

* Using this doesn't show message anymore when running locally on **localhost. (http://localhost:3000/)**

```
"start": "GENERATE_SOURCEMAP=false react-scripts start",
```

* Using this will show the source map message when running locally on **localhost. (http://localhost:3000/)**


```
"start": "react-scripts start",
```

**Comment by lalimasharda** [comment]
Hey @luan-dang-techlabs , this is a bug on our side and I will work on putting a fix out for it. Thanks!

**Comment by tnorling** [comment]
Addressed in #6398 and will be included in our next release

**Comment by zemzelett** [comment]
> Hey @luan-dang-techlabs , this is a bug on our side and I will work on putting a fix out for it. Thanks!

Sorry to bother but may i ask when you plan on releasing the fix for this?
